### ISSUE TITLE
HTMLInputElement::formNoValidate has the wrong type

### ISSUE THREAD

**Issue by sheley1998** [Microsoft/TypeScript#22367]
**TypeScript Version:**  2.7.1

**Search Terms:** 
formNoValidate

**Code**
```ts
const test = document.createElement('input');
test.formNoValidate = true;
```

**Expected behavior:**
compiler shouldn't complain since HTMLInputElement::formNoValidate is supposed to be a boolean value

**Actual behavior:**
compiler reports an error due to the fact that typescript treats HTMLInputElement::formNoValidate as a string instead of a boolean

**Playground Link:**  

**Related Issues:**

**Comment by mhegazy** [comment]
Should be fixed by https://github.com/Microsoft/TypeScript/pull/22381

**Comment by logme1** [comment]
thanks
